extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/WindowOperatorQueryKit.java

```diff
@@ -182,12 +244,10 @@ public QueryDefinition makeQueryDefinition(
   }
 
   /**
-   *
    * @param originalQuery
    * @param operatorList
-   * @return true if the operator List has a partitioning operator with an empty OVER clause, false otherwise
    */
-  private boolean ifEmptyOverPresentInWindowOperstors(
+  private void populateOperatorListFromQuery(
```

Comment:
this could now be `getOperatorListFromQuery` and return the operator list I guess as we are anyways sending a new array list instance in the variable

Reply:
Fair point. Will make the change.

Reply:
Done